keel-web/src/main/kotlin/com/netflix/spinnaker/config/DefaultConfiguration.kt

```diff
@@ -49,9 +51,11 @@ class DefaultConfiguration {
   fun idGenerator(): ULID = ULID()
 
   @Bean
+  @Primary
```

Comment:
I added this while tracking down what I thought might have been an issue with wiring our customized object mapper into Spring REST controllers, which wasn't an issue in the end. I don't think this hurts, but I can remove it if you feel I should. Let me know.